src/components/Footer/index.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <footer class="footer-container">
+      <h1>Footer Goes Here</h1>
+  </footer>
+</template>
+
+<script>
+export default {
+  name: 'Footer',
+};
+</script>
+
+<style scoped>
+.footer-container {
+    padding: 16px;
+    height: 200px;
+    background-color: #efefef;
```

Comment:
All colors should be moved to a colors.css file in the styles folder where you'll define the colors variables and import that file in other css files

Reply:
Ideally, we'll be theming the app with everything in the :root
We still have to decide on how we'd want to theme the application.
.
Making no changes here.